src/common/paths/buckets.yml

```diff
@@ -3,49 +3,120 @@ get:
   tags:
     - Buckets
   summary: List all buckets
+  description: |
+    Retrieves a list of [buckets]({{% INFLUXDB_DOCS_URL %}}/cloud/reference/glossary/#bucket).
+
+    To limit which buckets are returned, pass query parameters in your request.
+    If no query parameters are passed, InfluxDB returns all buckets up to the
+    default `limit`.
+
+    #### Limitations
+
+    - Paging with an `offset` greater than the number of records will result in
+    an empty list of buckets--for example:
+
+      The following request is paging to the 50th record, but the user only has
+      10 buckets.
+
+      ```sh
+      $ curl --request GET "INFLUX_URL/api/v2/scripts?limit=1&offset=50"
+
+      $ {
+          "links": {
+              "prev": "/api/v2/buckets?descending=false\u0026limit=1\u0026offset=49\u0026orgID=ORG_ID",
+              "self": "/api/v2/buckets?descending=false\u0026limit=1\u0026offset=50\u0026orgID=ORG_ID"
+          },
+          "buckets": []
+        }
+      ```
+
+    #### Related Guides
+
+    - [Manage buckets]({{% INFLUXDB_DOCS_URL %}}/cloud/organizations/buckets/)
   parameters:
     - $ref: "../parameters/TraceSpan.yml"
     - $ref: "../parameters/Offset.yml"
     - $ref: "../parameters/Limit.yml"
     - $ref: "../parameters/After.yml"
     - in: query
       name: org
-      description: The name of the organization.
+      description: |
+        Organization name.
+        The name of the organization.
       schema:
         type: string
     - in: query
       name: orgID
-      description: The organization ID.
+      description: |
+        Organization ID.
+        The organization ID.
```

Comment:
Depending on which applies, follow this: https://docs.influxdata.com/influxdb/cloud/api/#operation/PostWrite
or this: https://docs.influxdata.com/influxdb/cloud/api/#operation/GetTasks

I'm not sure `org` and `orgID` are used by Cloud at all here b/c they use the token organization to scope the results for most endpoints. We still need to test Cloud and OSS for this case-by-case.